adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.h

```diff
@@ -17,9 +17,14 @@
 
 typedef NS_ENUM(NSInteger, GADMintegralErrorCode) {
     /// Missing server parameters.
-    GADMintegralErrorInvalidServerParameters = 101
+    GADMintegralErrorInvalidServerParameters   = 101,
+    //The ad request was successful, but no ad was returned.
+    GADMintegralErrorAdNotFill = 102,
+    // The Mintegral SDK returned NO from its showAd call.
+    GADMintegralErrorAdNotValid  = 103
```

Comment:
Can you clarify what will cause this to happen?

Reply:
Can you clarify what will cause this to happen? 
102：The server may not be returning an ad
103：The returned ad data may exceed the valid cache time。

Does this apply to all ad formats?
102：Available for all ad types。
103：Use only with rewarded ad and interstitial ad